### PR TITLE
Format using Prettier.

### PR DESCRIPTION
**Pull request by harisvsulaiman**
### Use prettier to format agenda.
Based on the discussion at https://github.com/agenda/agenda/discussions/1209.

Note: Linting fails for now. I wanted to know what rules should be enforced on the code or should it be the prettier defaults?
cc @simison @simllll @OmgImAlexis

### CODE REVIEW

**Comment by koresar** [comment]
The most liveable mounting rules I know - the default built-in ESLint rules. We would not need to install, maintain additional dependencies.

**Comment by harisvsulaiman** [comment]
Now I am using eslint defaults along with prettier defaults.
The pre-commit hook via lint staged checks for issues

File: package.json [hunk 1]
```diff
@@ -57,75 +57,37 @@
...
     "delay": "4.4.0",
     "eslint": "7.18.0",
-    "eslint-config-xo": "0.34.0",
-    "eslint-config-xo-typescript": "0.37.0",
+    "eslint-config-prettier": "^7.2.0",
     "eslint-plugin-ava": "11.0.0",
     "eslint-plugin-eslint-comments": "3.2.0",
     "eslint-plugin-import": "2.22.1",
     "eslint-plugin-node": "11.1.0",
     "eslint-plugin-unicorn": "24.0.0",
```
Comment (simison) [inline package.json:71]: `eslint-plugin-unicorn` can be removed?

Comment (harisvsulaiman) [reply package.json:71]: Yes. You have to remove the eslint warning inside a file.

**Review by koresar — CHANGES_REQUESTED** [review]
.eslintcache needs to be removed and added to gitignore.

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["plugin:@typescript-eslint/recommended", "prettier"]
```
Comment (koresar) [inline .eslintrc:2]: Can we move this file contents to packege.json? Similar to other configurations.

File: package.json (hunk 1 above)
Comment (koresar) [reply package.json:71]: If we are using eslint defaults, we can pretty much remove most of the things which start with "eslint-".
Am I right?

Comment (harisvsulaiman) [reply package.json:71]: Yes. But eslint default are strict on "any" types.
I changed the error to a warning.
We can fix that after we merge this

File: .eslintrc (hunk 2 above)
Comment (harisvsulaiman) [reply .eslintrc:2]: I think the editor plugin for eslint picks up the config easily this way.what do you think @simison ?

**Review by harisvsulaiman — COMMENTED** [review]
Add warning for some defaults now.

File: .eslintrc (hunk 2 above)
Comment (koresar) [reply .eslintrc]: Editors pick up whatever config. Either in package.json, or in .eslintrc.

Comment (harisvsulaiman) [reply .eslintrc]: Ok.

**Review by koresar — APPROVED** [review]

**Comment by koresar** [comment]
I ran the master on my machine, then checked out the PR branch and tested it.
LGTM!
Thanks @harisvsulaiman

**Pull request merged**